### ISSUE TITLE
client request too frequent

### ISSUE THREAD

**Issue by developer-ken** [fffonion/Xunlei-Fastdick#161]
```
23:17:56 Initializing upgrade
23:18:01 Downstream error 711: client request too frequent
23:22:51 Initializing upgrade
23:22:56 Downstream error 711: client request too frequent
23:27:46 Initializing upgrade
23:27:52 Downstream error 711: client request too frequent
23:32:42 Initializing upgrade
23:32:47 Downstream error 711: client request too frequent
23:37:37 Initializing upgrade
23:37:42 Downstream error 711: client request too frequent
23:42:32 Initializing upgrade
23:42:37 Downstream error 711: client request too frequent
23:47:28 Initializing upgrade
23:47:33 Downstream error 711: client request too frequent
23:52:23 Initializing upgrade
23:52:28 Downstream error 711: client request too frequent
23:57:18 Initializing upgrade
23:57:23 Downstream error 711: client request too frequent
00:02:13 Initializing upgrade
00:02:18 Downstream error 711: client request too frequent
00:07:09 Initializing upgrade
00:07:14 Downstream error 711: client request too frequent
00:12:04 Initializing upgrade
00:12:09 Downstream error 711: client request too frequent
00:16:59 Initializing upgrade
00:17:59 Warning: error during downapi connection: <urlopen error timed out>, use portal: 47.103.83.42:12180
00:18:59 Warning: error during downapi connection: <urlopen error timed out>, use portal: 119.147.41.210:12180
00:18:59 Error: can't connect to down api
```
too short interval maybe?
this happens this week. It had worked well for quite a few weeks.

**Comment by developer-ken** [comment]
System: Debian 4.19.0
Using python.
江苏电信

**Comment by fffonion** [comment]
merged with #160
